ui/app/styles/components/generate-credentials-totp.scss

```diff
@@ -3,7 +3,8 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+@use '../utils/size_variables';
```

Comment:
this shouldn't be necessary, you can just call `$spacing-11` like before. Though I think we can delete this css file all together.